lib/commands.js

```diff
@@ -31,22 +39,28 @@ module.exports = {
     ],
 
     run: function(options) {
-      var config = this.project.config(options.environment);
-      var reportUri = options.reportUri;
+      let { environment, reportUri, silent } = options;
+      // TODO: find a way to get the build time configuration in Ember CLI command
+      let buildConfig = {};
```

Comment:
@rwjblue I would be glad for any hint how to get build time configuration in an Ember CLI command. I didn't found any way. :cry:

Reply:
What I've tried so far:

- Read the documentation of Ember CLI command and debugged `this`. I wasn't able to find any reference to app's build time configuration.
- Done a code search on EmberObserver for `includedCommands` and digged through the results. Wasn't able to find any addon that consumes app's build time configuration in an Ember CLI command.
- Construct path to app's `ember-cli-build.js` using `this.project.root` and pass it to [`ember-cli-build-config-editor`](https://github.com/srvance/ember-cli-build-config-editor). This one parses the file (AST) and provides a method to retrieve the configuration. But it does not support complex data structures like objects or arrays as config values. Therefore we can't use it as we have a very complex data structure for `policy` config option.
- Require app's `ember-cli-build.js` and initialize an instance of the app. This app should have the build time configuration as `options` property but it's always empty for some reason. This seems to depend on the `defaults` argument passed to the constructor - and I don't know how to mock that one.

I've no idea what options are left. Seems like a show stopper to me.

Reply:
> Require app's ember-cli-build.js and initialize an instance of the app. This app should have the build time configuration as options property but it's always empty for some reason. This seems to depend on the defaults argument passed to the constructor - and I don't know how to mock that one.

I think this is the right path for us at the moment. The argument passed and referred to as `defaults` is _essentially_ `{ project }` ([see here](https://github.com/ember-cli/ember-cli/blob/3575b9b8a96a392a20ae138290dd085667b579a7/lib/models/builder.js#L52)). 

Alternatively, we could move our config into a specific path that we can require (e.g. `config/content-security-policy.js`) instead of using `ember-cli-build.js` directly.

Reply:
I don't think initializing an instance of the app is an option here. I've investigated that one further and noticed that `ember-cli-build.js` returns a broccoli tree. It's `options` property is not populated with app's build options but with options passed to `broccoli-merge-trees`. Therefore it's an empty hash.

> Alternatively, we could move our config into a specific path that we can require (e.g. `config/content-security-policy.js`) instead of using `ember-cli-build.js` directly.

 This seems to be the path forward to me. Will implement it in another PR.

Reply:
Sounds good. I agree this seems like the most sensible option...

Reply:
@rwjblue Please review #104 for implementation.